### PR TITLE
Animal Search Update

### PR DESCRIPTION
**Pull request by SpencerjCarroll**
Updated the animal search to use Card Groups, modeled after the service request search.

### CODE REVIEW

**Comment by SpencerjCarroll** [comment]
This is ready for a review/feedback. We might want to make some adjustments to how we display the data but think the search is working how we want it to.

File: frontend/src/shelter/Shelter.js [hunk 1]
```diff
@@ -33,24 +33,24 @@ function Shelter() {
...
-      {data.shelters.map(shelter => (
-        <Link href={"/shelter/" + shelter.id}>
-          <ListGroup.Item action>{shelter.name} - {shelter.animal_count} Animals</ListGroup.Item>
-        </Link>
-      ))}
-    </ListGroup>
-    </>
-  )
-}
-export const NewShelter = () => (
```
Comment (AlexMountain) [inline frontend/src/shelter/Shelter.js:53]: This looks like a bad merge?  Can we revert to all of the previous changes here and just add in the `ANIMAL SEARCH` to it?

It also appears to be breaking the React server/tests

File: frontend/src/animals/AnimalTables.js [hunk 2]
```diff
@@ -40,40 +50,76 @@ export function AnimalSearch() {
...
+                  <ListGroup.Item className='owner'>{animal.owner ? <span><b>Owner:</b> {animal.owner_object.first_name} {animal.owner_object.last_name} <Link href={"/hotline/owner/" + animal.owner}> <FontAwesomeIcon icon={faClipboardList} inverse /></Link></span>  : ""}</ListGroup.Item>
+                  {/* <ListGroup.Item className='owner'>{animal.owner ? <span><b>Owner:</b> {animal.owner_object.first_name} {animal.owner_object.last_name} {animal.owner_object.phone} {animal.owner_object.first_name !== 'Unknown' ? <Link href={"/hotline/owner/" + animal.owner} className="mb-1" style={{width:23,height:23, minHeight:23, color:"#fff", backgroundColor: "#28a745"}} title="Owner details" aria-label="owner_details"><AssignmentIcon style={{fontSize:10}} /></Fab> : <Fab href={"/hotline/owner/edit/" + animal.owner} className="mb-1" style={{width:23,height:23, minHeight:23, color:"#fff", backgroundColor: "#28a745"}} title="Edit Owner" aria-label="edit_owner"><EditIcon style={{fontSize:10}} /></Fab>}</div> : ""}</ListGroup.Item> */}
```
Comment (AlexMountain) [inline frontend/src/animals/AnimalTables.js:116]: Cleanup (or maybe turn into the Shelter line?)

**Review by AlexMountain — COMMENTED** [review]
The search is super functional!  but the UI could still use some love.  Are you willing/interested in spending some time on it?

- [x] Header: Remove "Name: ".  Use "Unknown" if the animal doesn't have a name. Add species information here.
- [ ] Capitalization where appropriate and maybe combine into 1 row?  e.g. `"Large Adult M"`
- [ ] All information should still look good when optional information is missing.
- [ ] Add the animal serializer address somewhere.
- [ ] Add the animal front image somewhere.  Use the side image and lastly noimagefound as fallbacks.  see https://github.com/trevorskaggs/shelterly/blob/f4121916102aee2d9e39496242004d32a3a88530/frontend/src/people/PeopleViews.js#L99
- [ ] Add animal shelter name/clipboard link if exists.

**Comment by AlexMountain** [comment]
@SpencerjCarroll Looks like this still needs something to pass tests!

**Comment by SpencerjCarroll** [comment]
@AlexMountain think this is ready for a second look

**Review by AlexMountain — APPROVED** [review]
@SpencerjCarroll Made a few minor bug fixes, including one of my own from my most recent ticket merge!  If you approve of the changes then this is good to merge!  😼

**Pull request merged**